iex_trading/IEX_stocks.ipynb

```diff
@@ -294,30 +255,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The next function, `visualize_symbol` builds on the approach used in\n",
-    "`visualize_symbol_raster` above by overlaying each raster with the\n",
-    "appropriate x-range filtered (invisible) `Spikes` in order to enable\n",
-    "hovering.\n",
-    "\n",
-    "This is done using the same approach as the [previous\n",
-    "notebook](./IEX_trading.ipynb) where we use the `apply` method on the spikes\n",
-    "to apply the `xrange_filter function`. Note that as the `symbol`\n",
-    "argument changes, the `Spikes` object returned by the `select` method\n",
-    "also changes. This is why we need to set the `source` on our stream\n",
-    "dynamically.\n",
-    "\n",
-    "In addition, to keep everything consistent, we want to use our single\n",
-    "`range_stream` everywhere, including in the `DynamicMap` returned by\n",
-    "`spikes_aggregate`. This is done by passing `range_stream` explicitly in\n",
-    "the `streams` argument. This approach of using a single `RangeX` and\n",
-    "setting the `source` ensures that you can zoom in and then select a\n",
-    "different set of stocks to be displayed without resetting the zoom\n",
-    "level.\n",
-    "\n",
-    "\n",
-    "Lastly, we need to pass `expand=False` to prevent datashader from\n",
-    "filling the whole y-range (with NaN colors where there is no data) for\n",
-    "each raster generated."
+    "The next function, `visualize_symbol`, builds on the approach used in `visualize_symbol_raster` above by overlaying each raster with the appropriate x-range filtered (invisible) `Spikes` in order to enable hovering.\n",
+    "\n",
+    "This is done using the same approach as the [previous notebook](./IEX_trading.ipynb), where we use the `apply` method on the spikes to apply the `xrange_filter` function. Note that as the `symbol` argument changes, the `Spikes` object returned by the `select` method also changes. This is why we need to set the `source` on our stream dynamically.\n",
+    "\n",
+    "In addition, to keep everything consistent, we want to use our single `range_stream` everywhere, including in the `DynamicMap` returned by `spikes_aggregate`. This is done by passing `range_stream` explicitly in the `streams` argument. This approach of using a single `RangeX` and setting the `source` ensures that you can zoom in and then select a different set of stocks to be displayed without resetting the zoom level.\n",
+    "\n",
+    "Lastly, we need to pass `expand=False` to prevent Datashader from filling the whole y-range (with NaN colors where there is no data) for each raster genated.\n"
```

Comment:
```suggestion
    "Lastly, we need to pass `expand=False` to prevent Datashader from filling the whole y-range (with NaN colors where there is no data) for each raster generated.\n"
```

Reply:
Thanks, I'll make that change.